arviz/plots/pairplot.py

```diff
@@ -129,18 +132,18 @@ def pairplot(data, var_names=None, coords=None, figsize=None, textsize=None, kin
                     if i == j == 0 and colorbar:
                         hexbin = ax.hexbin(var1, var2, mincnt=1, gridsize=gridsize, **plot_kwargs)
                         divider = make_axes_locatable(ax)
```

Comment:
I can do this in a followup pr, but I'm not familiar with `make_axes_locatable`: could we move the `divider = make_axes_locatable(ax)` above the `if/else` block, and the `divider.append_axes('top' , size='7%').set_axis_off()` after the block?

Reply:
Honestly I don't know what this code does, I was focused on minimal changes but am happy to dive in.

What is the end result you're looking for so I can fix the problem?

Reply:
Just keeping code duplication to a minimum!

Reply:
Sounds good. Are you ok with merging this one and I'll follow up with your suggestions?

Reply:
Definitely - seems like the comments on here were all feature requests, rather than any problems with the code. We should merge when the tests pass.

Reply:
Thanks, I prone to a medical conditions where I have mild allergy to merge conflicts :)